chaincode/output.go

```diff
@@ -279,12 +279,13 @@ type outputModel struct {
 	Testtuple           outputTesttuple            `json:"testtuple"`
 }
 
-// TuplesEvent is the collection of tuples sent in an event
-type TuplesEvent struct {
+// Events is the collection of tuples sent in an event
+type Events struct {
 	Testtuples           []outputTesttuple           `json:"testtuple"`
 	Traintuples          []outputTraintuple          `json:"traintuple"`
 	CompositeTraintuples []outputCompositeTraintuple `json:"compositeTraintuple"`
 	Aggregatetuples      []outputAggregatetuple      `json:"aggregatetuple"`
+	ComputePlan          []outputComputePlan         `json:"computePlan"`
```

Comment:
Nit: The new property doesn't have an "s" but the existing properties do.

Reply:
Done